.gitmodules

```diff
@@ -10,3 +10,6 @@
 	path = nexa/eval/benchmark_tasks
 	url = https://github.com/NexaAI/benchmark-tasks.git
 	branch = main
+[submodule "dependency/bark.cpp"]
+	path = dependency/bark.cpp
+	url = https://github.com/NexaAI/bark.cpp.git
```

Comment:
add `branch=main` to solve the dependency issue

Reply:
For now it is indeed not depending on the bark.cpp main branch yet. I will switch to main once the bark.cpp change merged to main.